pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java

```diff
@@ -655,12 +655,12 @@ public void writeResourceQuotasToZooKeeper() throws Exception {
         // write the load factors
         setDynamicConfigurationToZK(LOADBALANCER_DYNAMIC_SETTING_LOAD_FACTOR_CPU_ZPATH, new HashMap<String, String>() {
             {
-                put(SETTING_NAME_LOAD_FACTOR_CPU, Double.toString(realtimeCpuLoadFactor));
+                put(SETTING_NAME_LOAD_FACTOR_CPU, Double.isFinite(realtimeCpuLoadFactor) ? Double.toString(realtimeCpuLoadFactor) : Double.toString(0.0));
```

Comment:
better to have a util function
```
double getFiniteValue(double value, double defaultValue) {
    return Double.isFinite(value) ? value : defaultValue;
}
```

so you can call `Double.toString(getFiniteValue(realtimeCpuLoadFactor, 0.0))`; and this util function can be reused